File: 编译原理/TinyScript-Js/src/parser/ast/Scalar.js
const Factor = require('./Factor')
const ASTNodeTypes = require('./ASTNodeTypes')

/**
 * 抽象语法树 - 标量节点
 */
class Scalar extends Factor {
  /**
   * 
   * @param {ASTNode} _parent
   * @param {PeekTokenIterator} it
   */
  constructor(_parent, it) {
    super(_parent, it)
  }
}

module.exports = Scalar

File: 编译原理/TinyScript/src/main/java/parser/ast/Scalar.java
package parser.ast;

import parser.util.PeekTokenIterator;

/**
 * 抽象语法树 - 标量节点
 */
public class Scalar extends Factor {
    public Scalar(ASTNode _parent, PeekTokenIterator it) {
        super(_parent, it);
    }
}
